pkg/kubelet/container_manager_linux.go

```diff
@@ -142,26 +142,48 @@ func createManager(containerName string) *fs.Manager {
 	}
 }
 
-const sysctlVmOvercommitMemory = "vm/overcommit_memory"
-
-// disableKernelMemoryOvercommitHandling tells the kernel to perform no memory over-commit handling.
-// Under this setting, the potential for memory overload is increased, but so is performance for
-// memory-intensive tasks
-// sets /proc/sys/vm/overcommit_memory to 1
-func disableKernelMemoryOvercommitHandling() error {
-	val, err := utilsysctl.GetSysctl(sysctlVmOvercommitMemory)
-	if err != nil {
-		return err
-	}
-	if val == 1 {
-		return nil
+// TODO: plumb this up as a flag to Kubelet in a future PR
+type KernelTunableBehavior string
```

Comment:
I'm working on stubbing out ContainerManager for integration tests as part of #14542. Can we do the same for local clusters?